doc: Updated release-notes-latest.rst for dev1 tag (crypto additions) (#3417)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-latest.rst

The following changes are relevant for all device families.

Crypto
------

* Added:

  * nrf_cc3xx_platform v0.9.5, with the following highlights:

    * Added correct TRNG characterization values for nRF5340 devices.

    See the :ref:`crypto_changelog_nrf_cc3xx_platform` for detailed information.
  * nrf_cc3xx_mbedcrypto version v0.9.5, with the following highlights:

>     * Built to match with the nrf_cc3xx_platform v0.9.5 including correct TRNG characterization values for nRF5340 devices.

Reviewer: ```suggestion
    * Built to match the nrf_cc3xx_platform v0.9.5 including correct TRNG characterization values for nRF5340 devices.
```

Author: Fixed

    See the :ref:`crypto_changelog_nrf_cc3xx_mbedcrypto` for detailed information.

* Updated:

>   * Rewrote nrf_security library stripping mechanism to not use cmake POST_BUILD stage (Was non-functional in certain versions of Segger Embedded Studio Nordic Edition).

Reviewer: ```suggestion
  * Rewrote the :ref:`nrfxlib:nrf_security`'s stripping mechanism to not use the POST_BUILD CMake stage.
    The previous code was non-functional in certain versions of SEGGER Embedded Studio (Nordic Edition).
```
Not sure if "Nordic Security Module's stripping mechanism" is correct, or if it should be "Nordic Security Module's library stripping mechanism"?

Author: Alternative wording committed

The following changes are relevant for all device families.

Crypto
------

* Added:

  * nrf_cc3xx_platform v0.9.5, with the following highlights:

    * Added correct TRNG characterization values for nRF5340 devices.

    See the :ref:`crypto_changelog_nrf_cc3xx_platform` for detailed information.
  * nrf_cc3xx_mbedcrypto version v0.9.5, with the following highlights:

    * Built to match the nrf_cc3xx_platform v0.9.5 including correct TRNG characterization values for nRF5340 devices.

    See the :ref:`crypto_changelog_nrf_cc3xx_mbedcrypto` for detailed information.

* Updated:

>   * Rewrote the :ref:`nrfxlib:nrf_security`'s library stripping mechanism to not use the POST_BUILD option in a custom build rule. The library stripping mechanism was non-functional in certain versions of Segger Embedded Studio Nordic Edition.

Reviewer: ```suggestion
  * Rewrote the :ref:`nrfxlib:nrf_security`'s library stripping mechanism to not use the POST_BUILD option in a custom build rule.
    The library stripping mechanism was non-functional in certain versions of SEGGER Embedded Studio Nordic Edition.
```
